packages/styles/src/components/form-search-input.scss

```diff
@@ -0,0 +1,98 @@
+@use './../functions/tokens';
+@use './../tokens/components';
+@use './../mixins/utilities';
+
+tokens.$default-map: components.$post-search-input;
+
+.search-input {
+  position: relative;
+
+  /* clears the ‘X’ from Internet Explorer */
+  input[type='search']::-ms-clear,
+  input[type='search']::-ms-reveal {
+    display: none;
+    width: 0;
+    height: 0;
+  }
```

Comment:
IE is dead anyway. If it's not also necessary in Edge, I would remove these lines.